src/query/service/src/interpreters/interpreter_table_modify_column.rs

```diff
@@ -182,39 +348,29 @@ impl Interpreter for ModifyTableColumnInterpreter {
 
         // NOTICE: if we support modify column data type,
         // need to check whether this column is referenced by other computed columns.
```

Comment:
Need to handle the warning: `need to check whether this column is referenced by other computed columns.`?
Also need a test for it.

Reply:
has test for this case, see:
https://github.com/datafuselabs/databend/pull/12139/files#diff-c9f542e7c99e6aecb18356ed72b1e30e567b9e11c219100c9dc2d1cadf86eeb0